src/main/java/com/pgs/client/ClientApplication.java

```diff
@@ -2,18 +2,11 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.context.annotation.Bean;
-import org.springframework.web.client.RestTemplate;
 
 @SpringBootApplication
 public class ClientApplication {
 
 	public static void main(String[] args) {
 		SpringApplication.run(ClientApplication.class, args);
 	}
-
-	@Bean
-	public RestTemplate getRestTemplate() {
```

Comment:
This bean should be actually configured in further way. We need basic authentication configuration.